bigquery/indexer.py

```diff
@@ -363,6 +388,7 @@ def create_mappings(es, index_name, table_name, fields, participant_id_column,
                     'ignore_above': 256
                 }
             }
+            properties[field_name]['analyzer'] = 'simple'
```

Comment:
// Use simple analyzer so underscores are treated as a word delimiter.
// Underscores in BQ column contents are not as common as underscores in column names, but
// some datasets have them (such as Baseline).